### PR TITLE
Fixes for globalsite search styles

### PR DESCRIPTION
**Pull request by OndoriSp**
- fix shadow styles for searchlist
- add shadows for search dropdown
- highlight found text in mainText

### CODE REVIEW

File: SportHub/src/modules/SearchDropdownList/components/SearchDropdownList.js [hunk 1]
```diff
@@ -124,15 +127,11 @@ const SearchDropdownList = ({ setIsContentSearch, setContentSearchValue }) => {
...
-          <Paper
-            style={{
-              width: "50vw",
-              maxWidth: "none",
-              marginTop: 3,
-              backgroundColor: "rgba(0, 0, 0, .0)",
-              borderRadius: 0,
-              boxShadow: "none"
-            }}>{children}</Paper>
+          showSuggestions && inputValue && articles.length !== 0 // removing shadows when nothing printed/no results
```
Comment (XashDM) [inline SportHub/src/modules/SearchDropdownList/components/SearchDropdownList.js:130]: can you refactor this into method showShadows() and move the comment to that method

File: SportHub/src/modules/SearchResultsList/components/SearchResultsList.js [hunk 2]
```diff
@@ -94,35 +96,37 @@ const SearchResultsList = ({ contentSearchValue }) => {
...
-                    :
-                    <></>
-                }
-                <span>{" > "}</span>
-                <span>{article.title}</span>
-              </li>
-              <li><span>{getMatchedMainText(article.mainText)}</span></li>
-            </ul>
+          <div className={styles.container}>
+            <div key={article.id} onClick={() => navigate(ROUTES.LOGIN)} className={styles.option}>
```
Comment (XashDM) [inline SportHub/src/modules/SearchResultsList/components/SearchResultsList.js:100]: Is this correct route?

**Review by XashDM — COMMENTED** [review]
Left some comments

File: SportHub/src/modules/SearchResultsList/components/SearchResultsList.js (hunk 2 above)
Comment (OndoriSp) [reply SportHub/src/modules/SearchResultsList/components/SearchResultsList.js:99]: There is no route for reading an article yet

File: SportHub/src/modules/SearchDropdownList/components/SearchDropdownList.js [hunk 3]
```diff
@@ -18,19 +18,27 @@ const SearchDropdownList = ({ setIsContentSearch, setContentSearchValue }) => {
...
+      setShowSuggestions(true)
     }
     setInputValue(value)
   }
 
   const handleArticlesGet = async () => {
-    if (inputValue) {
-      const result = await getSearchArticlesRequest(i18n.language, inputValue, 1, 3)
+    if (inputValue.trim()) { // preventing error when first symbol is space
+      const result = await getSearchArticlesRequest(i18n.language, inputValue, 1, 2)
```
Comment (Oleh-Kril) [inline SportHub/src/modules/SearchDropdownList/components/SearchDropdownList.js:28]: what's the magical numbers ?

File: SportHub/src/modules/SearchDropdownList/styles/style.module.scss [hunk 4]
```diff
@@ -1,4 +1,15 @@
+.optionsContainer {
+  max-height: 100%;
+  transform: translateX(-3rem) translateY(1.75vh);
```
Comment (Oleh-Kril) [inline SportHub/src/modules/SearchDropdownList/styles/style.module.scss:3]: why don't you use vw ?\

File: SportHub/src/modules/SearchResultsList/components/SearchResultsList.js [hunk 5]
```diff
@@ -94,35 +95,37 @@ const SearchResultsList = ({ contentSearchValue }) => {
...
+                        <span>{" > "}</span>
+                        <span>{article.team.teamName}</span>
+                      </>
+                      :
+                      <></>
+                  }
+                  <span>{" > "}</span>
+                  <span>{article.title}</span>
+                </li>
+                <li dangerouslySetInnerHTML={{ __html: getMatchedMainText(article.mainText) }}></li>
```
Comment (Oleh-Kril) [inline SportHub/src/modules/SearchResultsList/components/SearchResultsList.js:126]: is there are any other way to get around inserting InnerHTML ?

**Review by Oleh-Kril — APPROVED** [review]
consider comments and merge

**Pull request merged**